V2/server/model/tablequeries.js

```diff
@@ -3,12 +3,11 @@ import { Pool } from 'pg';
 
 dotenv.config();
 const pool = new Pool({ connectionString: process.env.DATABASE_URL });
```

Comment:
Can you create a different file where you will import pool from and connect to the db, in all other files that require pool, just import it in those files and re-use it. ie pool should only be imported once in the application